mu/logic.py

```diff
@@ -566,6 +566,15 @@ def load(self):
         """
         path = self._view.get_load_path(get_workspace_dir())
         logger.info('Loading script from: {}'.format(path))
+        # see if file is open first
+        for widget in self._view.widgets:
+            if widget.path is None:  # this widget is an unsaved buffer
+                continue
+            if path in widget.path:
+                self._view.show_message('Requested file is already open.')
```

Comment:
Something along the lines of 'The file "myprog.py" is already open' (without file path)
But that's getting petty over a single sentence in a solution to a problem we probably should have anticipated